clients/CMakeLists.txt

```diff
@@ -116,13 +116,41 @@ if( BUILD_CLIENTS_BENCHMARKS OR BUILD_CLIENTS_TESTS)
 
   # if it fails to find OpenMP compile and link flags in strange configurations it can just use non-parallel reference computation
   # if there is no omp.h to find the client compilation will fail and this should be obvious, used to be REQUIRED
-  find_package(OpenMP)
+  if( CMAKE_CXX_COMPILER MATCHES ".*/hipcc$" )
+    find_package(OpenMP)
```

Comment:
Similar question we shouldn't require hipcc to find_package(OpenMP) so can this just be moved two lines above?

Reply:
When the cmake compiler is hipcc `find_package` finds the correct ROCm libomp.so. Otherwise, gcc will get libgomp.so from gcc installation. I would rather get the `libomp` from ROCm to be consistent. With your current configuration cmake adds `libgomp.so` with `-L/rpaths` to the ROCm library. So I am fairly certain libgomp is ignored.